src/dev/src/app/quick-list/quick-list.demo.component.css

```diff
@@ -0,0 +1,6 @@
+h4 {
```

Comment:
this css file should not be needed, as we don't want to override the default headings and h4 is not even used in the demo